live-flowchart/sw.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * This is a service worker
+ */
```

Comment:
More comments will help to make the code the documentation for the project. See https://serviceworker-cookbook.herokuapp.com/docs/auto-claim/service-worker.html for documentation output.